v2/pkg/protocols/common/helpers/deserialization/testdata/java-deserialize.java

```diff
@@ -0,0 +1,30 @@
+import java.io.*;
+
+class Deserialize{  
```

Comment:
Normally if the file name and the class name does not match, the code will not compile.